fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStoreSplitRecordsTest.java

```diff
@@ -134,9 +134,9 @@ public void unsplitCompatibility(SplitRecordsTestConfig testConfig) throws Excep
         try (FDBRecordContext context = openContext()) {
             // Write a record using the old format
             recordStore = getStoreBuilder(context, simpleMetaData(NO_HOOK))
-                    .setFormatVersion(FDBRecordStore.SAVE_UNSPLIT_WITH_SUFFIX_FORMAT_VERSION - 1)
+                    .setFormatVersion(FormatVersion.FORMAT_CONTROL)
```

Comment:
It's probably fine either way, but one reason the tests were written the old way before is that they were trying to say "set the format version to the maximum that does _not_ support the save unsplit feature". That's a little bit less obvious with an explicit call to `FORMAT_CONTROL` like that. I'm not sure what you'd do instead, though, other than having either a `previous()` method on `FormatVersion`, or maybe introducing a local variable here:

```java
FormatVersion unsupportedVersion = FormatVersion.getFormatVersion(FormatVersion.SAVE_UNSPLIT_WITH_SUFFIX_FORMAT_VERSION.getValueForSerialization() - 1);
```